src/options.test.js

```diff
@@ -378,6 +378,21 @@ describe('options', () => {
     })
   })
 
+  describe('send args as option', () => {
+    it('should send args to jq execution', done => {
+      run('{"fruit":$myfruit,"fruit2":$myfruit2}', {}, {
+        input: 'json',
+        output: 'json',
+        args: { myfruit: { hello: 'orange' }, myfruit2: 'banana' }
+      })
+        .then(output => {
+          expect(output).to.eql({ fruit: { hello: 'orange' }, fruit2: 'banana' })
+          done()
+        })
+        .catch(e => done(e))
+    })
+  })
+
```

Comment:
Maybe it's worth adding a arg test case, also?

Reply:
My test case tests both , `arg` and `argjson` flags, when we pass `args` option. Should I test it separately?

Reply:
I added a few more comprehensive test cases to test all combinations. This should work.